src/main/cli/settings.c

```diff
@@ -199,7 +199,7 @@ static const char * const lookupTableGyro[] = {
 
 #ifdef USE_GPS
 static const char * const lookupTableGpsProvider[] = {
-    "NMEA", "UBLOX", "MSP"
+    "UBLOX", "MSP"
```

Comment:
With this we broke the backward compatibility with MSP:
https://github.com/betaflight/betaflight/blob/96e6b794307e7aab3090ae89f70c219308f33ee8/src/main/msp/msp.c#L1507-L1508

Reply:
NMEA will be removed in configurator.

Reply:
Yes, I understand that, so we are going to drop support in old versions of Configurator and from external MSP clients if some exist.

Reply:
@McGiverGim You requested removing NMEA on Discord. I have added configurator change. Do we need any additional changes?

Reply:
No, simply I wanted to know if we are going to drop the backward support. Until now we maintain backward support always to not break old MSP clients (including Configurator, but not only Configurator).
But if we have decided to drop it, I simply will approve this. I was waiting for a statement or group decision saying yes to this decision.
And thanks for this code, of course.

Reply:
I think generally we maintained backwards support, and I think that we should do so in this case, unless it would be difficult.